services/figma/README.md

```diff
@@ -0,0 +1,8 @@
+# Figma
+
+[![Figma's Logo][producti]][product]
+
+Collaborative design tool. We have started using this on some new projects.
```

Comment:
The "started using this on some new projects" bit seems like it could become outdated. Maybe a designer would have some info/different language that could be added here.